workflows/conversion/conversion/config.vsh.yaml

```diff
@@ -0,0 +1,46 @@
+functionality:
+  name: "conversion"
+  namespace: "conversion"
+  description: "A pipeline to convert different file formats to .h5mu."
+  authors:
+    - name: Dries Schaumont
+      email: dries@data-intuitive.com
+      roles: [ author, maintainer ]
+      props: { github: DriesSchaumont, orcid: "0000-0002-4389-0440" }
+  argument_groups:
+    - name: Inputs
```

Comment:
Can you add an output argument for choosing the output filename?

Reply:
Done!